frontend/src/components/homepage/sections/HomepageHero.tsx

```diff
@@ -16,10 +16,10 @@ const Hero = () => {
     >
       <GridContainer className="hero--container padding-y-1 tablet:padding-y-3 tablet-lg:padding-y-10 desktop-lg:padding-top-15 desktop-lg:padding-bottom-10 position-relative">
         <div className="hero--text position-relative z-100">
-          <h1 className="text-ls-neg-2 tablet:font-sans-2xl desktop-lg:font-sans-3xl desktop-lg:margin-top-2 text-balance">
-            <span>{t("pageTitle")}</span>
+          <h1 className="text-ls-neg-2 line-height-sans-2 font-sans-xl margin-top-3 tablet:font-sans-2xl desktop-lg:font-sans-3xl desktop-lg:margin-top-2 desktop-lg:margin-bottom-5">
+            {t("pageTitle")}
           </h1>
-          <p className="usa-intro line-height-sans-3 font-sans-md tablet:font-sans-lg text-balance">
+          <p className="text-balance font-sans-md margin-y-2 tablet:font-sans-lg tablet:margin-y-3">
```

Comment:
- Remove unnecessary typography and margin classes
- Remove unnecessary `span`
- Still need responsive font size classes so the hero looks good at various sizes